src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Behaviors/Animations/AnimationTypes/TadaAnimation_Tests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.CommunityToolkit.UnitTests.Mocks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Behaviors.Animations.AnimationTypes
+{
+	public class TadaAnimation_Tests
+	{
+		[SetUp]
+		public void SetUp() => Device.PlatformServices = new MockPlatformServices();
+
+		[Test]
+		public async Task AFullAnimationShouldReturnToOriginalValues()
+		{
+			var animation = new TadaAnimation();
+			var view = new MockView();
+
+			await animation.Animate(view);
+
+			Assert.IsTrue(view.ValuesSet.ContainsKey(nameof(View.Scale)));
+
+			// Assert.AreEqual(1, view.ValuesSet[nameof(View.Scale)].Last());
+
+			Assert.IsTrue(view.ValuesSet.ContainsKey(nameof(View.Rotation)));
+
+			// Assert.AreEqual(0, view.ValuesSet[nameof(View.Rotation)].Last());
```

Comment:
Let's not leave commented code in. 

We should either uncomment or delete this line.